sql/moz-fx-data-shared-prod/telemetry/unified_metrics/metadata.yaml

```diff
@@ -2,6 +2,13 @@ friendly_name: Unified Metrics
 description: |-
   Unified view on common browser metrics across all products.
 
+  Note. From Q4/2022 the app name includes the suffix ` Glean`
```

Comment:
The downstream dependencies are [updated to query the view in this story](https://mozilla-hub.atlassian.net/jira/software/c/projects/DENG/boards/465?modal=detail&selectedIssue=DENG-433&quickFilter=2143).

Before back-filling, requires a notification in the #data channel for queries from other sources. E.g. Jupyter notebook, Redash.

Reply:
It might be wise to follow the [communication guidelines here](https://mozilla-hub.atlassian.net/wiki/spaces/DATA/pages/6848559/Communicating+changes) to communicate this.

Reply:
Thanks a lot for this link!

Reply:
Sent the communication.